README.md

```diff
@@ -265,17 +265,6 @@ const { data: app } = await requestWithAuth(
       Name of previews, such as `mercy`, `symmetra`, or `scarlet-witch`. See <a href="https://developer.github.com/v3/previews/">API Previews</a>.
     </td>
   </tr>
-  <tr>
-    <th align=left>
-      <code>options.method</code>
-    </th>
-    <td>
-      String
-    </td>
-    <td>
-      Any supported <a href="https://developer.github.com/v3/#http-verbs">http verb</a>, case insensitive. <em>Defaults to <code>Get</code></em>.
-    </td>
-  </tr>
```

Comment:
wait why was method removed?

Reply:
It was not intentional/part of the final changeset, just inflight drafting...